src/util/crypto.ts

```diff
@@ -122,38 +151,39 @@ function reconstruct(shardObjects, passphrase) {
     } provided`;
   }
 
-  var nonces = shardObjects.map(shard => shard.nonce);
+  const nonces = shardObjects.map(shard => shard.nonce);
   if (!nonces.every(n => n === nonces[0])) {
     throw "Nonces mismatch among shards!";
   }
 
-  var titles = shardObjects.map(shard => shard.title);
+  const titles = shardObjects.map(shard => shard.title);
   if (!titles.every(t => t === titles[0])) {
     throw "Titles mismatch among shards!";
   }
 
-  var versions = shardObjects.map(shard => shard.version);
+  const versions = shardObjects.map(shard => shard.version);
   if (!versions.every(v => v === versions[0])) {
     throw "Versions mismatch along shards!";
   }
 
   switch (versions[0]) {
     case 0:
-      var shardData = shardObjects.map(shard => shard.data);
-      var encryptedSecret = SECRETS.combine(shardData);
-      var secret = dehexify(encryptedSecret);
-      var nonce = dehexify(nonces[0]);
-      var salt = hashString(titles[0]);
+      var shardData = shardObjects.map(shard => shard.data),
```

Comment:
Usually we start each declaration with keyword. But it's on your linter. 
And is `var` ok?

Reply:
> Usually we start each declaration with keyword

Yeah, that's the matter of preference. Staring each declaration with keyword seems excessively verbose. But if that's the common practice, I'll change it back.

P.S. Currently linter has no rules about it.